components/navbar.html

```
@@ -7,57 +7,41 @@
+     onmouseover="this.style.background='#00BFFF'; this.style.color='white';"
+     onmouseout="this.style.background='transparent'; this.style.color='white';">
+    <span>Contribution</span>
+  </a>
+
+  <a href="website/pages/contact.html" class="navbar-link" data-page="contact"
+     style="position: relative; display: inline-flex; justify-content: center; align-items: center; padding: 0.5rem 0.8rem; border-radius: 6px; transition: background 0.3s ease, color 0.3s ease; color: white;"
+     onmouseover="this.style.background='#00BFFF'; this.style.color='white';"
+     onmouseout="this.style.background='transparent'; this.style.color='white';">
+    <span>Contact</span>
+  </a>
+</div>
```

HTMLHint (1.9.1)

[error] Tag must be paired, missing: [ </nav> ], open tag match failed [ <nav class="glassmorphism-navbar"> ] on line 1.

(tag-pair)